javascript/statements.json

```diff
@@ -1022,7 +1022,7 @@
               "version_added": "6"
             },
             "nodejs": {
-              "version_added": true
+              "version_added": "0.1.100"
```

Comment:
This one looks questionable. Chrome didn't get it until 49, but it was supported from Node's first release? I'm guessing the Chrome data is here the one that's the problem, but I think these should be mutually sensical before merging.

Reply:
Great finding! I think the for-in support might have been confused here with for-of support. I've added a commit that sets it to the first version of browsers given this dates all the way back to ECMAScript 1.